### DISCUSSION TITLE
我创建了一个QQ群, 欢迎大家一起来交流讨论

### DISCUSSION THREAD

**Discussion by sqzxcv** [photoprism/photoprism#1333] [Show and tell]
![image](https://user-images.githubusercontent.com/3991441/119853518-db203580-bf42-11eb-84d8-d4a574159fb8.png)

**Comment by sqzxcv** [comment]
QQ群号: 487931699

**Comment by graciousgrey** [comment]
?

**Comment by sqzxcv** [comment]
> ?

this is a group for Chinese user

**Comment by graciousgrey** [comment]
Ok, I will move this to discussions as this is not a to do :)
